db/migrate/20171124131432_create_docker_registries.rb

```diff
@@ -0,0 +1,11 @@
+class CreateDockerRegistries < ActiveRecord::Migration
+  def change
+    create_table :docker_registries do |t|
+      t.string :url
+      t.boolean :mirror
```

Comment:
I think `mirror` needs to be a string for the remote registry. If `nil` then not a mirror. Any other idea?

Reply:
i guess any mirror is also a docker registry, so this field just specifies if it's intended as a mirror or not? that would mean we could keep it that way

Reply:
How are you gonna relate the mirror registry with the remote registry without knowing? Remember the daemon.json that we need to produce in the end.

Reply:
I agree with @vitoravelino, but instead I'd say that if it's not `null` then it's a mirror on `url` mirroring `mirror`. If `mirror` is null, it's a registry.